static/src/javascripts/projects/journalism/modules/submit-form.js

```diff
@@ -43,28 +44,52 @@ const showWaiting = cForm => {
     });
 };
 
+const readFile = file =>
+    new Promise(res => {
+        const reader = new FileReader();
+        reader.addEventListener(
+            'load',
+            () => {
+                // remove 'data:*/*;base64,' from the start of the string
+                const imageAsBase64 = reader.result
```

Comment:
Do we have a mask on the type of files that can be uploaded? It may be useful in some situations:
```html
<input type=file accept="image/png,image/gif,image/jpeg">
```